package.json

```diff
@@ -25,6 +25,7 @@
         "@angular/platform-browser-dynamic": "16.2.3",
         "@angular/router": "16.2.3",
         "@angular/service-worker": "16.2.3",
+        "ngx-slider-v2": "16.0.2",
```

Comment:
Yes because sliders e.g. used for the competency thresholds are just standard inputs of type range. I initially tried to use them as well but they did not provide enough configuration options for my needs.

Reply:
please talk to us beforehand, in this particular case it is ok